docs/content/latest/secure/tls-encryption/connect-to-cluster.md

```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
 
 ## Prerequisites
 
-Before you can enable and use server-to-server encryption, you need to create and configure server certificates for each node of your YugabyteDB cluster. For information, see [Create client certificates](../client-certificates).
+Before you can use client certificates to connect to your YugabyteDB clusters, server-to-server encryption and client-to-server encryption must be enabled. For details, see [Enable server-to-server encryption](../server-to-server/) and [Enable client-to-server encryption](../client-to-server).
 
-For each client that will connect to a YugabyteDB cluster, you need the following three files to be accessible on the client computer.
+Also, you need to create the required client certificates. For details, see [Create client certificates](../client-certificates).
 
-- `root.crt` — root certificate file
-  - To generate, see [Prepare root configuration file](../prepare-nodes/#generate-root-config)
-- `yugabytedb.crt` — private node certificate
-  - To generate, see [Generate private key for each node](../prepare-nodes/#generate-private-key-for-each-node)
-- `yugabytedb.key` — private node key
-  - To generate, see [Generate private key for each node](../prepare-nodes/#generate-private-key-for-each-node)
+Each client that connects to a YugabyteDB cluster needs the following files to be accessible on the client computer.
+
+- `ca.crt` — root certificate file (YSQL and YCQL)
```

Comment:
Did you confirm that naming the file as `ca.crt` works for YSQL?

Also, I still find this a bit hard to read (with YCQL and YSQL mentioned for every step), but it's okay to handle it later.